app.rb

```diff
@@ -145,6 +149,28 @@ def config
 
     private
 
+    def get_ial_select_options(is_eipp_allowed:)
+      enhanced_ipp_option = ['enhanced-ipp-required', 'Enhanced In-Person Proofing (Enabled in dev & staging only)']
+      option = [
+        ['1', 'Authentication only (default)'],
+        ['2', 'Identity-verified'],
+        ['0', 'IALMax'],
+        ['step-up', 'Step-up Flow'],
+        ['biometric-comparison-required', 'Biometric Comparison (Disabled in prod)'],
+      ]
+
+      if is_eipp_allowed
+        option.push(enhanced_ipp_option)
+      else
+        option
+      end      
+    end 
+
+    def is_eipp_allowed()
+      env = ENV['idp_environment']
+      is_eipp_allowed = (env == 'dev' || env == 'staging')
+    end
```

Comment:
1. I'd drop the `is_` and name this with a `?` to match the previous comment
2. No need for the final assignment, the result is just the value
```suggestion
    def eipp_allowed?
      env = ENV['idp_environment']
      env == 'dev' || env == 'staging'
    end
```

Reply:
Another thought: hardcoding specific environment names in source code is a [12-factor app antipattern](https://12factor.net/config#:~:text=In%20a%20twelve,over%20its%20lifetime.), could we add a new config for `ENV['eipp_allowed']` and set that in the appropriate environments?

Reply:
Thank you for the recommendation. I think that is a better solution. Resolved with commit db8c6741ec7d24e4524248238bab00ae45f1fbc4.